{{cookiecutter.repostory_name}}/README.md

```diff
@@ -100,6 +100,45 @@ If one wants to deploy other branch, force may be used to push desired branch to
 ```sh
 git push --force production local-branch-to-deploy:master
 ```
+{% if cookiecutter.use_allauth == 'y' %}
+# External auth (OAuth, OpenID connect etc.)
```

Comment:
please make this section collapsible; see  https://github.com/reef-technologies/cookiecutter-rt-django/pull/191/files#diff-378103025370115268a1f51cd31dc9694912f0739345121e4d6f3c473097b7d6R27  for example